packages/perseus/src/widgets/radio/score-radio.ts

```diff
@@ -10,17 +12,15 @@ function scoreRadio(
     rubric: PerseusRadioRubric,
     strings: PerseusStrings,
 ): PerseusScore {
+    const validationError = validateRadio(userInput);
+    if (validationError) {
+        return validationError;
+    }
+
     const numSelected = userInput.choicesSelected.reduce((sum, selected) => {
         return sum + (selected ? 1 : 0);
     }, 0);
 
-    if (numSelected === 0) {
-        return {
-            type: "invalid",
-            message: null,
-        };
-    }
-
```

Comment:
The remaining validation checks rely on data in the scoringData object, so I don't think we can move those to the validation function. If it's possible, would love to know so we can move all that over!